lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/BinarySpec.hs

```diff
@@ -240,6 +247,17 @@ spec = do
             res <- try' (runGet getAddress (BL.fromStrict $ unsafeFromHex hex))
             res `shouldSatisfy` isRight
 
+        describe "golden block0s from jcli" $ do
```

Comment:
Ok, `jcli` is probably not the best term here. We are using an instance in `jormungandr-lib`, so "jormungandr-lib" would probably be more appropriate.

Reply:
you produced them using the rust test in `jormungandr-lib` as specified in  `lib/jormungandr/test/data/block0s/jormungandr.patch ` above. So I believe the description should go :
`golden block0s generated from jormungandr-lib`

Reply:
changed in https://github.com/input-output-hk/cardano-wallet/pull/633/commits/40766eeeef457a9fa4c9f0a641df6b9f02a9219e